packages/theia-integration/src/node/glsp-socket-server-contribution.ts

```diff
@@ -153,6 +161,12 @@ export abstract class GLSPSocketServerContribution extends BaseGLSPServerContrib
 
     protected override processLogInfo(line: string): void {
         if (line.startsWith(START_UP_COMPLETE_MSG)) {
+            const port = this.getPortFromStartupMessage(line);
```

Comment:
Should we also add error case handling here?
Meaning, if a fixed port  (>0) is configured via options should we check that the actual used port is equal to the defined fixed port?

Reply:
Good idea! I will do that.